app/services/monitoring/AnnualAccountingAuditBlockBuilder.scala

```diff
@@ -37,7 +37,7 @@ class AnnualAccountingAuditBlockBuilder @Inject()() {
               "paymentMethod" -> details.paymentMethod,
               "annualStagger" -> returns.staggerStart,
               "paymentFrequency" -> details.paymentFrequency,
-              "estimatedTurnover" -> eligibilitySubmissionData.estimates.turnoverEstimate,
+              "estimatedTurnover" -> returns.turnoverEstimate.getOrElse(BigDecimal(eligibilitySubmissionData.estimates.map(_.turnoverEstimate).getOrElse(0l))),
```

Comment:
Instead of .getOrElse(0l) it'd be better to make it throw an exception inside the getOrElse as we always have the turnover